app/shell-window/ui/promptbar.js

```diff
@@ -55,7 +56,7 @@ export function remove (page, prompt) {
   if (!page.prompts) { return } // page no longer exists
 
   // find and remove
-  var i = page.prompts.indexOf(prompt)
+  var i = findIndex(page.prompts, prompt)
```

Comment:
I don't know how this was working previously?

Reply:
Why's that?

Reply:
I played with it and saw it not finding my prompt. Could be I did something wrong, like hoping it to match based on the objects in there rather than a memory address for an actual object.